apps/openmw/mwclass/container.cpp

```diff
@@ -239,7 +239,8 @@ namespace MWClass
         info.caption = ref->mBase->mName;
 
         std::string text;
-        if (ptr.getCellRef().getLockLevel() > 0)
+        int lockLevel = ptr.getCellRef().getLockLevel();
+        if (lockLevel > 0 && lockLevel != std::numeric_limits<int>::max())
```

Comment:
why not just use -1 ?

Reply:
-1 is a negative level. Locks with negative levels are Unlocked.

Reply:
Derp, ok.